str/associatr/pseudobulk.py

```diff
@@ -83,6 +90,7 @@ def pseudobulk(input_file_path, target_sum, min_pct):
 
 # inputs:
 @click.option('--input-dir', help='GCS Path to the input AnnData object')
+@click.option('--sample-id-file-path', help='GCS Path to the sample ID file to do the analysis on')
```

Comment:
This script is now 9 args, it's worth considering a config file. Relates to other comment on whether adding these SG IDs in config is reasonable

Reply:
ok i added a config